### DISCUSSION TITLE
Team member invite email not coming

### DISCUSSION THREAD

**Discussion by mir1198yusuf** [plausible/analytics#3788] [Self-Hosted Support] [answered]
Hello All, I followed this exact steps and did nothing special for smtp, still emails are not coming.
git clone https://github.com/plausible/hosting
cd hosting
openssl rand -base64 64 | tr -d '\n' ; echo
added secret key and base url in env file
docker-compose up -d
installed nginx standalone (not in compose file), and did required nginx config (proxy_pass and x-forwarded-for header)
setup tls on nginx using certbot
everything working fine just that email are not coming
checked logs of mailer container , found these : 
`Connecting to aspmx.l.google.com [2404:6800:4003:c1c::1b]:25 ... failed: Cannot assign requested address` when i added a team member to invite to plausible

kept everytime default for smtp, in other words, did not touched this part https://plausible.io/docs/self-hosting-configuration#mailersmtp-setup at all

EC2 outbound all traffic open, 25 open for inbound too

Please let me know , what's missing here

**Comment by ruslandoga** [comment]
👋 @mir1198yusuf 

> EC2 outbound all traffic open, 25 open for inbound too

Note that outgoing connections to port 25 are blocked on all new EC2 instances, regardless if the port is open in the security groups.

Please see https://repost.aws/questions/QUhOiYfU2ESzyjHNFLdih5lQ/smtp-port-25-outgoing-connections-blocked

You might be able to open it by filing a ticket with AWS support team.

**Reply by mir1198yusuf** [reply]
@ruslandoga the error `cannot assign requested address` means the same & is due to this AWS issue only ?

**Reply by mir1198yusuf** [reply]
@ruslandoga what is the solution for this ? using mailer container as relay for ses SMTP configuration?

**Reply by ruslandoga** [reply, marked as answer]
The solution is to either use AWS SES or file a support ticket to open port 25.
In the case of AWS SES you don't need the `mail` container.

**Reply by mir1198yusuf** [reply]
@ruslandoga thanks for the reply, I thought the error means having some network connectivity issue from inside container

**Reply by mir1198yusuf** [reply]
@ruslandoga you mentioned I don't need the mail container meaning i just add the SMTP config here in this envs https://plausible.io/docs/self-hosting-configuration#mailersmtp-setup

this seems much better way
I was going to add container & then relay to smtp

**Reply by ruslandoga** [reply]
Sorry for a late reply. Yes, just setting the SMTP env vars in Plausible to point to AWS SES should work fine :)
